chainer/functions/activation/sigmoid.py

```diff
@@ -66,13 +66,13 @@ def sigmoid(x, use_cudnn=True):
     Args:
         x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
         :class:`cupy.ndarray`):
-            Input variable. A :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+            Input variable. A :math:`(s_1, s_2, ..., s_N)`-shaped float array.
         use_cudnn (bool): If ``True`` and cuDNN is enabled, then this function
             uses cuDNN as the core implementation.
 
     Returns:
         ~chainer.Variable: Output variable. A
-        :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+        :math:`(s_1, s_2, ..., s_N)`-shaped float array.
 
     .. admonition:: Example
 
```

Comment:
How about adding an example to show the concrete values of the outputs? For example,

```
It maps the input values into the range of :math:`[0, 1]`.

>>> x = np.random.uniform(-3, 3, (3,)).astype('f')
>>> x
array([-2.43635702,  0.45567897,  2.57577729], dtype=float32)
>>> F.sigmoid(x).data
array([ 0.08044198,  0.6119886 ,  0.92928624], dtype=float32)
```

And before the below line (L.79), how about adding an explanation line to say the shape consistency like:

```
It is an element-wise operation, so the shape of the output array is the same as the input array.
```